### PR TITLE
cmd/tailscale: show an error when we hit the multiple users Android bug

### PR DESCRIPTION
**Pull request by andrew-d**
Updates tailscale/tailscale#2180

Signed-off-by: Andrew Dunham <andrew@du.nham.ca>

### CODE REVIEW

**Comment by andrew-d** [comment]
Note: I don't have an Android device to test on, so I would appreciate someone else helping with that 😃

File: cmd/tailscale/backend.go [hunk 1]
```diff
@@ -81,6 +81,11 @@ var googleDNSServers = []netip.Addr{
 // VPN status was revoked.
 var errVPNNotPrepared = errors.New("VPN service not prepared or was revoked")
 
+// errMultipleUsers is used when we get a "INTERACT_ACROSS_USERS" error, which
+// happens due to a bug in Android. See:
+//    https://github.com/tailscale/tailscale/issues/2180
+var errMultipleUsers = errors.New("VPN cannot be created on this device due to an Android bug with multiple users")
```
Comment (bradfitz) [inline cmd/tailscale/backend.go:87]: Include some short link users can go to for more info?

Like `tailscale.com/s/multi-user-bug` ?

That can just be a www redirects.txt entry to some open or closed bug in oss.

Comment (bradfitz) [reply cmd/tailscale/backend.go:87]: Whoops, I meant to put this comment on the const multipleUsersText below, not this error.

**Review by bradfitz — APPROVED** [review]
Code seems fine, but haven't run it.

Does it compile at least? :)

File: cmd/tailscale/backend.go (hunk 1 above)
Comment (andrew-d) [reply cmd/tailscale/backend.go:87]: Done in https://github.com/tailscale/tailscale-www/pull/1610 - good idea

**Comment by andrew-d** [comment]
> Code seems fine, but haven't run it.
> 
> Does it compile at least? :)

Yep, compiles when I run `make dockershell` and `make tailscale-debug.apk` inside the container.

**Pull request merged**